### ISSUE TITLE
part_seg on my own dataset

### ISSUE THREAD

**Issue by Rancho86** [charlesq34/pointnet2#137]
I would like to use the part_seg net of pointnet++ to train on my own dataset. I change the data in the file "/data/shapenetcore...normal/", 
 
![YX(8}11VF31LI0@6)Q CMQ4](https://user-images.githubusercontent.com/31204036/61276582-ea7cb680-a7e2-11e9-9bff-ceb1f7290f18.jpg)
the format of the dataset is same as the example.
![~A%UJ}X R223@}YO6$V2F45](https://user-images.githubusercontent.com/31204036/61276729-2adc3480-a7e3-11e9-9c98-37f1a6551888.png)

the content in the file train_test_split 
![微信图片_20190716152826](https://user-images.githubusercontent.com/31204036/61276609-f4061e80-a7e2-11e9-8dde-2111947d6156.jpg)
![微信图片_20190716152830](https://user-images.githubusercontent.com/31204036/61276611-f4061e80-a7e2-11e9-95e1-5cdfe4aca5dc.jpg)
![微信图片_20190716152835](https://user-images.githubusercontent.com/31204036/61276614-f49eb500-a7e2-11e9-80b0-7972747ff97a.jpg)

and synsetoffset2category.txt. 
![微信图片_20190716152821](https://user-images.githubusercontent.com/31204036/61276621-f7010f00-a7e2-11e9-9b5a-636bebc94d4a.jpg)

In the meanwhile, I change the seg_classes in the file ”seg_part/part_dataset_all_normal.py“ ，NUM_CLASSES in the file"seg_part/train.py". But it is reported the bug. 
![7A416EC817FE0950788C3BD89D34639D](https://user-images.githubusercontent.com/31204036/61276631-fbc5c300-a7e2-11e9-8deb-75835772d507.jpg)
This question has puzzled me for a long time, please help me answer it.

**Comment by ZJU-PLP** [comment]
@Rancho86 Hi, did you solve this problem yet? Could you mind sharing your method of this issue? Thanks a lot!

**Comment by wx0205** [comment]
Hi, what did you use to label your dataset? Thank you!

**Comment by mactavish10** [comment]
In the test.py file, change num_batches from num_batches = pc.shape[0]/batch_size to num_batches = pc.shape[0]/batch_size/2 . That might work, it worked for me.
Also, could you elaborate on how you trained it on your own dataset?

**Comment by lyy-1997-hub** [comment]
兄弟，你跑通分类了没，我只能跑通h5格式的modelnet40_ply_hdf5_2048，还找不到办法跑通modelnet40_normal_resampled，help！！！！
